remote_protocol.gen.go

```diff
@@ -191,7 +191,7 @@ type NodeGetMemoryStats struct {
 // DomainDiskError is libvirt's remote_domain_disk_error
 type DomainDiskError struct {
 	Disk string
-	Error int32
+	remote_error int32
```

Comment:
I looked into this change. The generator isn't doing the right thing here - you wouldn't expect a parameter _named_ "Error" to be renamed "remote_error". This is because the code generator passes both identifier names and type names into the same routine to be transformed. There are other symptoms of this in the generated code today: for example, a number of struct fields get renamed from "Secret" to "OptSecret", which is a type name.

I do want to fix this problem, but as it predates your PR and will be an API-breaking change (because some public struct field names will change), I'll do it as a separate PR after yours has merged.